web/nginx/app.conf.template

```diff
@@ -63,4 +63,8 @@ server {
     proxy_cache_bypass $http_upgrade;
     proxy_redirect off;
   }
+
+  location /monitoring {
```

Comment:
lets move this above beside the /spark and /alloy location blocks. it will keep it consistent and just in case there is any precedence rules

Reply:
moved between alloy and spark